screen-orientation/onchange-event.html

```diff
@@ -1,35 +1,26 @@
 <!DOCTYPE html>
+<meta charset="utf-8" />
+<meta viewport="width=device-width, initial-scale=1" />
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="/resources/testdriver.js"></script>
 <script src="/resources/testdriver-vendor.js"></script>
-<script>
+<script type="module">
+import { makeCleanup } from "./resources/orientation-utils.js";
 promise_test(async t => {
-  t.add_cleanup(async () => {
-    try {
-      await document.exitFullscreen();
-    } catch (e) {}
-    screen.orientation.unlock();
-  });
-  await test_driver.bless("request full screen", () => {
-    return document.documentElement.requestFullscreen();
-  });
+  t.add_cleanup(makeCleanup());
+  await test_driver.bless("request full screen");
+  await document.documentElement.requestFullscreen();
```

Comment:
I'm not sure this change is correct, ditto for the change below.

See: https://web-platform-tests.org/writing-tests/testdriver.html#test_driver.bless

requestFullscreen requires user activation in WebKit at least, so we need to keep that as it was

Reply:
It will be ok. It works as follows:

```JS
// this creates a button, and test_driver clicks it, then awaits resolves. 
await test_driver.bless("request full screen");
// We now have transient/user activation
await document.documentElement.requestFullscreen();
```

Reply:
Don't want to block this, but this is because the unprefixed version in WebKit uses a slightly different codepath that avoids the user activation logic, but I realize the callback version won't necessarily work either, since the button would have already been clicked at this point